src/state/actions/regular.ts

```diff
@@ -10,6 +9,7 @@ import * as buddyApi from '../../api/buddies';
 import * as messageApi from '../../api/messages';
 import * as statApi from '../../api/stat';
 import { PollingParams } from '../reducers/messages';
+import { UpdateKey } from '../reducers/updateMentorData';
```

Comment:
too generic for this namespace maybe? also it kinda sucks that we are using two styles of importing here.

Reply:
Yeah, good point.